bokehjs/src/lib/polyfill.ts

```diff
@@ -23,6 +28,43 @@ if (typeof Object.values === "undefined") {
   }
 }
 
+if (typeof Uint8Array.prototype.fill === "undefined") {
+  (Uint8Array.prototype as any).fill = Array.prototype.fill;
+  (Int8Array.prototype as any).fill = Array.prototype.fill;
+  (Uint16Array.prototype as any).fill = Array.prototype.fill;
+  (Int16Array.prototype as any).fill = Array.prototype.fill;
+  (Uint32Array.prototype as any).fill = Array.prototype.fill;
+  (Int32Array.prototype as any).fill = Array.prototype.fill;
+  (Float32Array.prototype as any).fill = Array.prototype.fill;
+  (Float64Array.prototype as any).fill = Array.prototype.fill
```

Comment:
@mattpap missing semicolon here and below?

Reply:
No. This is the minimal number of semicolons needed here, otherwise this code would be interpreted as:
```ts
(Uint8Array.prototype as any).fill = Array.prototype.fill(Int8Array.prototype as any).fill = Array.prototype.fill ...
```
Linter would complain if the last semicolon was added.